src/components/YearlyDataTab.tsx

```diff
@@ -65,11 +73,22 @@ export default class YearlyDataTab extends React.Component<YearlyDataTabProps, Y
     }
     this.setState({...this.state, ...{ yearsToShow }});
   }
+  toggleMonths() {
+    this.setState({...this.state, ...{months: !this.state.months}});
```

Comment:
You can just do ` this.setState({months: !this.state.months});`, similar to `toggleFormatting`. `setState` should already be doing the merging of the entire state with the attribute you want to update.

Reply:
I think eventually having a `toggleStateVal(stateKey)` function would be helpful to reduce all the functions that only toggle a specific key in the sate.